src/main/java/at/hannibal2/skyhanni/features/mining/crystalhollows/MetalDetectorAllToolsAlert.kt

```diff
@@ -0,0 +1,47 @@
+package at.hannibal2.skyhanni.features.mining.crystalhollows
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.data.MiningApi
+import at.hannibal2.skyhanni.events.ActionBarUpdateEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.ItemUtils.getInternalName
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NeuInternalName.Companion.toInternalName
+import kotlin.time.Duration.Companion.seconds
+
+@SkyHanniModule
+object MetalDetectorAllToolsAlert {
+    private val METAL_DETECTOR = "DWARVEN_METAL_DETECTOR".toInternalName()
+    private val DWARVEN_LAPIS_SWORD = "DWARVEN_LAPIS_SWORD".toInternalName()
+    private val DWARVEN_EMERALD_HAMMER = "DWARVEN_EMERALD_HAMMER".toInternalName()
+    private val DWARVEN_GOLD_HAMMER = "DWARVEN_GOLD_HAMMER".toInternalName()
+    private val DWARVEN_DIAMOND_AXE = "DWARVEN_DIAMOND_AXE".toInternalName()
+
+    private val config get() = SkyHanniMod.feature.mining.metalDetector.metalDetectorAllToolsAlert
+
+
```

Comment:
```suggestion
    private val config get() = SkyHanniMod.feature.mining.metalDetector.metalDetectorAllToolsAlert

```
remove extra empty line